contrib/inventory/freeipa.py

```diff
@@ -3,8 +3,9 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 import argparse
-from ipalib import api
 import json
+from ipalib import api, errors
+from ansible.module_utils.six import u
```

Comment:
If you are going to utilize `ansible.module_utils`, I'd recommend `ansible.module_utils._text.to_text`.  However, since this is in contrib, we have a history of not utilizing `ansible.module_utils` here, so just using `from six import u` is probably best.